wasp/src/main/java/com/orhanobut/wasp/Wasp.java

```diff
@@ -189,6 +189,9 @@ public Wasp build() {
         }
 
         private void init() {
+            if (endPointUrl == null) {
```

Comment:
init can return a boolean value to show if it is successful or not. if we are throwing an error, then app would crash here